src/bun.js/ConsoleObject.zig

```diff
@@ -170,6 +183,7 @@ pub fn messageWithTypeAndLevel(
                 tabular_data,
                 properties,
             );
+            table_printer.value_formatter.indent += console.default_indent;
```

Comment:
nit: `TablePrinter.init()` should accept current default indentation as a parameter. right now it's always initialized to 0